tests/test_project.py

```diff
@@ -601,13 +601,16 @@ def test_update_submodules_list(repos_tmpdir):
 
     # Make tagged_repo to be zephyr project submodule.
     subprocess.check_call(
-        [GIT, 'submodule', 'add', str(tagged_repo), 'tagged_repo'], cwd=zephyr)
+        [GIT, '-c', 'protocol.file.allow=always',
```

Comment:
Could you add some new `SUBMODULE_ADD_OPTS` constant to avoid the repetition and make testing local changes easier?

Reply:
> Could you add some new SUBMODULE_ADD_OPTS

Done for this change and the subsequent one testing the related west update option.